alexrallen/sonic-buildimage
# [mellanox] Remove redundant breakout mode for SN3800

#### Why I did it

With https://github.com/Azure/sonic-buildimage/pull/9278 we no longer need to specify breakout modes with a different default speed separately. 

#### How I did it

Removed discrete 1x10G breakout mode from SN3800 platform.json file

#### How to verify it

Apply the Mellanox-SN3800-D28C49S1 SKU and verify all ports are configured as expected.

#### Which release branch to backport (provide reason below if selected)

The mentioned fix was backported to 202111 so I figure this should be backported as well to be aligned. 
https://github.com/Azure/sonic-buildimage/commit/546818f41442f3377cadc23e1089a958f0683562


#### Description for the changelog
[mellanox] Remove redundant breakout mode for SN3800

#### A picture of a cute animal (not mandatory but encouraged)
![tigris](https://user-images.githubusercontent.com/5898707/151867342-9b068ce1-e01c-4e95-993b-13cefb4ba611.jpg)
